routing/routing_tests/index_graph_test.cpp

```diff
@@ -401,15 +401,23 @@ UNIT_TEST(OneSegmentWay)
   traffic::TrafficCache const trafficCache;
   shared_ptr<EdgeEstimator> estimator = CreateEstimatorForCar(trafficCache);
   unique_ptr<WorldGraph> worldGraph = BuildWorldGraph(move(loader), estimator, vector<Joint>());
-
-  auto const start = IndexGraphStarter::MakeFakeEnding(
-      Segment(kTestNumMwmId, 0, 0, true /* forward */), m2::PointD(1, 0), *worldGraph);
-  auto const finish = IndexGraphStarter::MakeFakeEnding(
-      Segment(kTestNumMwmId, 0, 0, true /* forward */), m2::PointD(2, 0), *worldGraph);
-
   vector<Segment> const expectedRoute(
       {{kTestNumMwmId, 0 /* featureId */, 0 /* seg id */, true /* forward */}});
-  TestRoute(start, finish, expectedRoute.size(), &expectedRoute, *worldGraph);
+
+  // Starter must match any combination of start and finish directions.
+  vector<bool> const tf = {true, false};
```

Comment:
To prevent warnings, please, double braces, i.e.:
```c++
{{true, false}};
```

Reply:
done